### PR TITLE
Add methods to save info sistematically

### PR DESCRIPTION
**Pull request by fvalle1**
Add function to simplify the process of saving the output

### CODE REVIEW

File: sbmtm.py [hunk 1]
```diff
@@ -523,13 +524,35 @@ def group_to_group_mixture(self,l=0,norm=True):
...
             return n_td_tw
 
-    def pmi_td_tw(self,l=0):
-        '''
-        Point-wise mutual information between topic-groups and doc-groups, S(td,tw)
-        This is an array of shape Bd x Bw.
-
-        It corresponds to
-        S(td,tw) = log P(tw | td) / \tilde{P}(tw | td) .
+    def plot_topic_dist(self, l):
```
Comment (martingerlach) [inline sbmtm.py:548]: The new function plot_topic_dist seems to partially remove the function pmi_td_tw

**Review by martingerlach — CHANGES_REQUESTED** [review]

File: sbmtm.py (hunk 1 above)
Comment (martingerlach) [reply sbmtm.py:548]: Thanks for the addition. 
Perhaps we can add it without altering the existing functions?

**Comment by fvalle1** [comment]
Sure, I’ll check and avoid to touch existing function!

sorry..

Filippo

File: sbmtm.py (hunk 1 above)
Comment (fvalle1) [reply sbmtm.py:548]: should be fixed now

**Review by martingerlach — APPROVED** [review]

**Comment by martingerlach** [comment]
Great. thanks for the addition.

**Pull request merged**